src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/PluginImpl.java

```diff
@@ -139,7 +150,7 @@ public Api getApi() {
     public static PluginImpl getInstance() {
         Jenkins jenkins = Jenkins.getInstance();
         if (jenkins != null) {
-            return jenkins.getPlugin(PluginImpl.class);
+            return GlobalConfiguration.all().get(PluginImpl.class);
```

Comment:
can't you do this?
```suggestion
            return ExtensionList.lookupSingleton(PluginImpl.class);
```

Reply:
There are places where powermock is used to mock the response of the static method. So I can do that, but it would require changing more test code, and depending on (power)mocking API out of the plugin source base. Which I choose to avoid.